### PR TITLE
Add sentis library tag

### PR DESCRIPTION
**Pull request by pauldog**
To add the unity-sentis library tag to sentis models.

### CODE REVIEW

File: packages/tasks/src/library-ui-elements.ts
Comment (pcuenca) [inline packages/tasks/src/library-ui-elements.ts:788]: This doesn't exist (the `ml-agents` example above is a fork, and it's going to be replaced in #439. Maybe you'd wnat to use https://github.com/Unity-Technologies/sentis-samples instead?

File: packages/tasks/src/library-ui-elements.ts
Comment (pcuenca) [inline packages/tasks/src/library-ui-elements.ts:548]: Where is the C# file provided? In the model repo?

File: packages/tasks/src/library-ui-elements.ts
Comment (pcuenca) [inline packages/tasks/src/library-ui-elements.ts:786]: I'd maybe use `unity-sentis` for consistency with the tag. This is how it's displayed in the case of `ml-agents`:

Screenshot 2024-01-17 at 20 11 00

File: packages/tasks/src/library-ui-elements.ts
Comment (simoninithomas) [reply packages/tasks/src/library-ui-elements.ts:788]: Yes it can be interesting to add a Sentis documentation, as we did with ML-Agents: https://huggingface.co/docs/hub/ml-agents

File: packages/tasks/src/library-ui-elements.ts
Comment (pauldog) [reply packages/tasks/src/library-ui-elements.ts:548]: Hi yes, we provide it as a single file in the repo.

File: packages/tasks/src/library-ui-elements.ts
Comment (pauldog) [reply packages/tasks/src/library-ui-elements.ts:788]: Hi, yes I can change the github link to the samples link. (I wasn't sure if that github gets created automatically... apparently not) .

It would be good to have Sentis documentation (how do we do this?). We have our own documentation too: https://docs.unity3d.com/Packages/com.unity.sentis@latest

Comment (pcuenca) [reply packages/tasks/src/library-ui-elements.ts]: If you can provide the text that's awesome 🔥 We can add a page similar to the one @simoninithomas mentioned and link from there to your existing docs.

File: packages/tasks/src/library-ui-elements.ts
Comment (Wauplin) [inline packages/tasks/src/library-ui-elements.ts:545]: Out of curiosity, what is the "model name" referred to here? Can it be inferred from the repo_id on the Hub? Asking because if you can infer it from `ModelData`, that would make the snippet even better. In `ModelData` class you have access to the model id, the tags, the config, etc.

(if not possible, just ignore my comment :) )

Comment (pauldog) [reply packages/tasks/src/library-ui-elements.ts:545]: Hi, unfortunately this can't be inferred from the ModelData as in a lot of situations (e.g. Whisper or Stable Diffusion) it requires multiple *.sentis files. So I think the best we can do is give a general example here. We also provide a full C# code along side each model which most people will use instead of the snippet. 

I think there are two other option which might be better?: (1) Just remove this snippet entirely. (2) Do some fancy code which outputs the entire c# file as the snippet - which is probably overkill.

Comment (Wauplin) [reply packages/tasks/src/library-ui-elements.ts:545]: Thanks for the reply @pauldog! In the meantime I saw the discussion you've already had on slack about it so I'll defer to them for what's the best solution.

File: packages/tasks/src/library-ui-elements.ts
Comment (pauldog) [reply packages/tasks/src/library-ui-elements.ts]: I put together[ some text for the doc page](https://docs.google.com/document/d/1Er5PFY5oy9j_OWeQKjyhLrZZyiTWPM1HsRsQUwR_BdM/edit?usp=sharing)

Is this OK? Do we add it ourselves or do you add it? Thanks.

**Comment by simoninithomas** [comment]
LGTM I let @osanseviero have the last word on it

File: packages/tasks/src/library-ui-elements.ts
Comment (osanseviero) [inline packages/tasks/src/library-ui-elements.ts:546]:
```suggestion
Model model = ModelLoader.Load(Application.streamingAssetsPath + "/" + modelName);
```

File: packages/tasks/src/library-ui-elements.ts
Comment (osanseviero) [reply packages/tasks/src/library-ui-elements.ts]: You can open a PR directly to `hub-docs` such as this file https://github.com/huggingface/hub-docs/blob/main/docs/hub/asteroid.md

**Review by osanseviero — APPROVED** [review]
LGTM after the suggested change

**Pull request merged**
